userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -0,0 +1,150 @@
+---
+sidebar_position: 1
+---
+
+# Installing with GitOps
+
+## Environment setup
+
+To install profiles with GitOps you will need to have completed the following environment setup.
+
+ :white_check_mark: [Cluster](/docs/tutorial-basics/setup#kubernetes-cluster)
+
+ :white_check_mark: [Pctl binary](/docs/tutorial-basics/setup#pctl-the-profiles-cli)
+
+ :white_check_mark: [Profiles CRDs and Flux CRDs](/docs/tutorial-basics/setup#profiles-crds-and-flux-crds)
+
+ :white_check_mark: [GitHub repo](/docs/tutorial-basics/setup#a-github-repo-synced-to-flux)
+
+ :white_check_mark: [GitHub token](/docs/tutorial-basics/setup#personal-access-token)
+
+The full setup docs can be found [here](/docs/tutorial-basics/setup#prerequisites)
+
+## Simple install from a profile URL
+
+:::caution Private repos
+If either your GitOps repo or the repo containing the profile you wish to install
+are private, remember to ensure that your local git environment is configured correctly.
+:::
+
+To install a profile, we use `pctl install`. To see all flags available on this subcommand,
+see [the help](/docs/pctl/install).
+
+There are two methods by which you can install a profile: with a direct URL and via a catalog.
+Please see [the page here](/docs/installer-docs/using-catalogs) for specific instructions on how to install a profile from a catalog.
+
+With the following command `pctl` will:
+- generate a set of manifests for each artifact declared in the profile at the given URL
+- commit those manifests to a branch in your GitOps repo
+- push that branch and
+- open a PR on your GitOps repo to merge the changes.
+Your GitOps repo is the one you synced to Flux in your cluster in the
+[environment setup](/docs/tutorial-basics/setup#a-github-repo-synced-to-flux).
+
+```bash
+pctl install \
+  --profile-url <URL of profile to install> \
+  --create-pr \
+  --pr-repo <gitops repo username or orgname>/<gitops repo name>
+```
+
+Above we use the following flags:
+- `--profile-url`. This is the full git URL of the profile you wish to install on your cluster.
+- `--create-pr`. This directs `pctl` to open a PR against the main branch of your GitOps repo.
+  _Note that this flag is only supported for GitHub._
+- `--pr-repo`. The partial URL of your GitOps repo synced to your cluster, in the format
+  `username/repo-name`.
+
+:::info
+We recommended also setting the `--git-repository` flag. See [below](#the-git-repository-flag)
+for more information.
+:::
+
+Once you have run the command, navigate to your GitOps repo and approve the PR.
```

Comment:
```suggestion
Once you have run the command, navigate to your GitOps repo and approve and merge the PR.
```

Reply:
Kind of would like to point out that just approving doesn't automatically merge it, since we don't set auto-merge flag.